docs/src/standalone/pages/vegetation/radiative_transfer/beer_model.md

```diff
@@ -0,0 +1,59 @@
+# Radiative transfer scheme
+This section describes multiple models of radiative transfer 
+through the vegetation canopy, implemented in ClimaLSM. 
+
+## Beer's law
+Plants utilize Photosynthetically Active Radiation (PAR) for the process of photosynthesis, during which they convert light energy into chemical energy, fueling the synthesis of sugars and other organic compounds. PAR refers to the portion of the electromagnetic spectrum that is essential for photosynthesis in plants. PAR includes wavelengths ranging from approximately 400 to 700 nanometers and corresponds to the visible light spectrum. The unit used to measure PAR is called micromoles per square meter per second (μmol/m²/s), representing the number of photons within the PAR range that strike a square meter of a surface per second.
+
+The portion of PAR that is actually absorbed by the vegetation canopy for photosynthesis is called Absorbed Photosynthetically Active Radiation (APAR). The APAR driving photosynthesis is calculated following the Beer-
+Lambert law:
+
+```math
+APAR(PAR, \theta_s) = (PAR)(1 - \rho_{leaf})(1 - e^{(-K(\theta_s) LAI  \Omega)})
+```
+
+where PAR ≈ SW/2 is the incident moles of photons per meter squared per
+second in the PAR window, approximated as half of the incident shortwave flux.
```

Comment:
I think this might be supposed to read "approximated as half of the incident shortwave flux if PAR is not directly available. $p_{leaf}$ is the ...."

I think this change in the punctuation makes it make a little more sense?